src/app/code/community/SchumacherFM/Markdown/Model/Observer/Adminhtml/Block.php

```diff
@@ -179,7 +179,7 @@ protected function _addMarkDownConfig()
 
         $config['stores']  = $this->_helper->getStoreCodes();
         $config['eeloc']   = $this->_helper->isEpicEditorLoadOnClick();
-        $config['hideIIB'] = $this->_helper->isHiddenInsertImageButton();
+        $config['hideIIB'] = ($this->_helper->isHiddenInsertImageButton()?true:'false');
```

Comment:
Would you please explain why on the one hand you need a bool true value and on the other hand you are outputting a string 'false' value?